klippa/src/lib.rs

```diff
@@ -10,19 +11,25 @@ mod gvar;
 mod hdmx;
 mod head;
 mod hmtx;
+mod inc_bimap;
 mod layout;
 mod maxp;
 mod name;
+mod offset;
+mod offset_array;
 mod os2;
 mod parsing_util;
 mod post;
 pub mod serialize;
 mod stat;
+mod variations;
+use inc_bimap::IncBiMap;
 pub use parsing_util::{
-    parse_drop_tables, parse_name_ids, parse_name_languages, parse_unicodes, populate_gids,
+    parse_name_ids, parse_name_languages, parse_tag_list, parse_unicodes, populate_gids,
```

Comment:
just noticing now, but are these methods genuinely part of the public API?

Reply:
oh this is because I need to use these methods in integration_test, do you know how can I export them to integration_test without making them public? I tried `pub(crate) use` but that doesn't work.

Reply:
yea integration test only works on public API.